coordinator/src/services/translation/request/index.ts

```diff
@@ -12,13 +12,24 @@ import {InvalidValueError} from "../../../models/errors/processing-errors"
 import {convertHL7V3DateTimeToIsoDateTimeString} from "../common/dateTime"
 import * as hl7V3 from "../../../models/hl7-v3"
 import * as fhir from "../../../models/fhir"
-import {translateReleaseRequest} from "./dispense/release"
+import {convertDispenseNotification} from "./prescription/prescription-dispense"
+import {translateReleaseRequest} from "./dispensation/release"
 
 export function convertBundleToSpineRequest(bundle: fhir.Bundle, messageId: string): SpineRequest {
   const messageType = identifyMessageType(bundle)
-  return messageType === fhir.EventCodingCode.PRESCRIPTION
-    ? requestBuilder.toSpineRequest(createParentPrescriptionSendMessagePayload(bundle), messageId)
-    : requestBuilder.toSpineRequest(createCancellationSendMessagePayload(bundle), messageId)
+  let createPayloadFunction: () => hl7V3.SendMessagePayload<unknown>
+  switch (messageType) {
+    case fhir.EventCodingCode.PRESCRIPTION:
+      createPayloadFunction = () => createParentPrescriptionSendMessagePayload(bundle)
+      break
+    case fhir.EventCodingCode.CANCELLATION:
+      createPayloadFunction = () => createCancellationSendMessagePayload(bundle)
+      break
+    case fhir.EventCodingCode.DISPENSE:
```

Comment:
I think it's worth waiting until that ticket is raised rather than guessing how we're going to approach it and then change it again? I've simplified this method as you suggested in an earlier comment, did you want to check the change and see if it reads any better?